src/app/layouts/layout.ejs

```diff
@@ -127,6 +127,11 @@
                 <li><a href="<%=locals.urls.interactions%>/terms">Terms and conditions</a></li>
                 <li><a href="<%=locals.urls.help%>/contact">Need help using this service?</a></li>
                 <li><a href="<%=locals.urls.interactions%>/privacy">Privacy Notice</a></li>
+                <li class="govuk-footer__inline-list-item">
```

Comment:
Do you need to add a `class` here as I cannot see in other footer items above like `<li><a href="<%=locals.urls.interactions%>/terms">Terms and conditions</a></li>` ?

Reply:
I have removed it as this class is not necessary because this page is using the old GDS. 
The page needs to be updated with the new design in a separate ticket.